### ISSUE TITLE
MIxS overlay for DataHarmonizer AND Portal?

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#254]
In the 2022-02-02 Metadata meeting, we were talking about displaying alternative descriptions for the MIxS environmental triad terms in the Portal. @cmungall said that could be implemented as an overlay file.  

To some degree that kind of this is already implemented in the [Soil-NMDC-Template_Compiled](https://docs.google.com/spreadsheets/d/1pSmxX6XGOxmoA7S7rKyj5OaEl3PmAl4jAOlROuNHrU0/edit#gid=1894635793) Google Sheet, which has a concept of MIxS modified columns.

Let's make sure we don't have parallel solutions for this.

**Comment by turbomam** [comment]
> @turbomam Is this ticket still relevant with the transition to sheets and friends?

Yeah, it think it is. I think of it as being more nmdc_schema related than sheets_and_freinds related. Although it does address the fact that any modifications you've made in a Google Sheet (to configure a DataHarmonizer) could be reflected in the nmdc-schema. I'll be addressing this as part of the move from the static MIxS 5 import to a dynamic MIxS import.

**Comment by mslarae13** [comment]
> > @turbomam Is this ticket still relevant with the transition to sheets and friends?
> 
> Yeah, it think it is. I think of it as being more nmdc_schema related than sheets_and_freinds related. Although it does address the fact that any modifications you've made in a Google Sheet (to configure a DataHarmonizer) could be reflected in the nmdc-schema. I'll be addressing this as part of the move from the static MIxS 5 import to a dynamic MIxS import.

Sounds great! Let me know if you need anything from me. But I do agree. The updates and added terms will result in schema updates

**Comment by mslarae13** [comment]
using structured annotations for help menus
